### ISSUE TITLE
There are incorrect tests

### ISSUE THREAD

**Issue by LuuchoRocha** [nmaguirre/minesweeper#58]
### In MinesweeperCellTest:
```
 @Test
 public void isBlockedTest() {
     MinesweeperCell cell = new MinesweeperCell();
     cell.putMine();
     assertTrue(cell.isBlocked());
 }
```
It should check if ` assertTrue(cell.hasMine())`

### In MinesweeperGameStateTest:
```
@Test
public void gameEndedTest () {
     MinesweeperGameState gameState = new MinesweeperGameState();
     assertTrue(gameState.gameEnded());
}
```

It should check `assertFalse(gameState.gameEnded())`

**Comment by LuuchoRocha** [comment]
Fixed in 0776bdc225482620f63420e537f2bb6e91041ca8

**Comment by nricci** [comment]
Well done
